winml/lib/Api/LearningModel.cpp

```diff
@@ -21,13 +21,23 @@ LearningModel::LearningModel(
   _winmlt::TelemetryEvent loadModel_event(_winmlt::EventCategory::kModelLoad);
 
   WINML_THROW_IF_FAILED(CreateOnnxruntimeEngineFactory(engine_factory_.put()));
-  auto file_handle = wil::unique_handle(CreateFileW(path.c_str(),
-                                        GENERIC_READ,
-                                        0,
-                                        NULL,
-                                        OPEN_EXISTING,
-                                        FILE_ATTRIBUTE_READONLY,
-                                        NULL));
+
+  wil::unique_handle file_handle{
+#if WINVER >= _WIN32_WINNT_WIN8
+      CreateFile2(path.c_str(),
```

Comment:
Why CreateFile2 is better?

Reply:
The other one is unavailable in UWP

Reply:
A bunch of APIs are removed from UWP to "encourage" use of the newer or safer variants

Reply:
Is CreateFile2 not available downlevel, can we just use that?

Reply:
It's not available